package.json

```diff
@@ -17,6 +17,7 @@
     "@nuxtjs/svg": "^0.4.0",
     "@sentry/tracing": "^6.19.7",
     "core-js": "^3.19.3",
+    "newrelic": "^9.9.0",
```

Comment:
hello, @marsellavaleria19 should we remove sentry from the package list?

Reply:
Hello, @Ibwedagama .. Yes, we should remove this. But, i forgot to remove this.  Thank you for reminding me. This is my commit : https://github.com/jabardigitalservice/super-app-web/pull/33/commits/75b23933ef2f730f6e6ffe25d22ec968c438247b